docs/iis.md

```diff
@@ -13,3 +13,52 @@ The following table provides the list of metrics that the Ops Agent collects fro
 | agent.googleapis.com/iis/new_connection_count            | cumulative | 1    |        | Connections opened to IIS. |
 | agent.googleapis.com/iis/request_count                   | cumulative | 1    | state  | Requests made to IIS. |
 
+# `iis_access` Logging Receiver
+
+## Configuration
+
+To configure a receiver for your IIS system logs, specify the following fields:
+
+| Field                 | Default                           | Description |
+| ---                   | ---                               | ---         |
+| `type`                | required                          | Must be `iis_access`. |
+| `include_paths`       | `[C:\inetpub\logs\LogFiles\W3SVC1\u_ex*]` | A list of filesystem paths to read by tailing each file. A wild card (`*`) can be used in the paths; for example, `C:\inetpub\logs\LogFiles\W3SVC1\u_ex*`. |
```

Comment:
One more tiny nit: I forgot that YAML doesn't like `:` appearing without quotes so when `C:` got added this became an invalid string -- it should be quoted instead:

```suggestion
| `include_paths`       | `['C:\inetpub\logs\LogFiles\W3SVC1\u_ex*']` | A list of filesystem paths to read by tailing each file. A wild card (`*`) can be used in the paths; for example, `C:\inetpub\logs\LogFiles\W3SVC1\u_ex*`. |
```